examples/http-request-errorbudget.yaml

```diff
@@ -0,0 +1,37 @@
+recordingrule:
+- expr: |
+    sum(label_replace(increase(promhttp_metric_handler_requests_total{namespace="default",job="fooapp"}[30d]), "status_class", "${1}xx", "code", "([0-9])..")) by (status_class)
+  labels:
+    job: fooapp
+    namespace: default
+  record: status_class:promhttp_metric_handler_requests_total:increase30d
+- expr: |
+    (
+      0.001000
+    *
+      sum(status_class:promhttp_metric_handler_requests_total:increase30d)
+    )
+  labels:
+    job: fooapp
+    namespace: default
+  record: promhttp_metric_handler_requests_errorbudget_requests
```

Comment:
I would put the `30d` in here somehow. This is now not following the Brazilian naming conventions (which might be fine in this situation), so I cannot say how exactly, but you could imagine that somebody wants different timespans, and then it would be good to have the relevant timespan here in the metric name.

Reply:
Couldn't really come up with a proper naming for this metric when adding the window.
`promhttp_metric_handler_requests_errorbudget_requests30d` or `promhttp_metric_handler_requests_errorbudget30d_requests` both feel weird. 
What do you think about surfacing a `window: 30d` label? :thinking:

Reply:
The `window` label has the problem that it breaks the golden rule of label dimensions being aggregatable is some way. I think it is fine on alerts, as alerts have “informational” labels anyway (like `severity`). In recording rules, it is quite common to put the range in the name somewhere.

Reply:
In this case, keeping the `:increase30d` suffix would make sense, IMHO. It's still an increase over 30d. (Your error count is allowed to increase by this amount over 30d.)